UnityProject/Assets/Light2D/Examples/Editor/EditorCameraWindow.cs

```diff
@@ -1,20 +1,20 @@
-﻿using UnityEngine;
-using UnityEditor;
+﻿using UnityEditor;
+using UnityEngine;
 
 public class EditorCameraWindow : EditorWindow
 {
-    string myString = "Hello World";
-    bool groupEnabled;
-    bool myBool = true;
-    float myFloat = 1.23f;
+    private bool groupEnabled;
+    private bool myBool = true;
+    private float myFloat = 1.23f;
+    private string myString = "Hello World";
```

Comment:
Are these dummy variables really in use?

Reply:
I will detlete all light2d examples. They shouldn't be in our codebase anyway.
Those are used within light2d example suite, which is bullshit